ONSdigital/rasrm-acceptance-tests
# Add respondent home system test

# Motivation and Context

Adds the simple healtcheck system test for respondent home ui. 

# How to test?



- Clone and checkout the docker branch linked in the PR below
- `make pull && make up` in the docker directory (should bring up a tagged version of respondent home ui - a Python app)
- Run the system tests with behave: `pipenv run behave system_tests/features`

# Links



- ~[Add to docker PR](https://github.com/ONSdigital/ras-rm-docker-dev/pull/65)~ (now on master)
- [Python app PR](https://github.com/ONSdigital/respondent-home-ui/pull/22)
